### PR TITLE
fixes hulks attacking with dual eswords

### PR DESCRIPTION
**Pull request by PKPenguin321**
Fixes https://github.com/tgstation/tgstation/issues/18293

### CODE REVIEW

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -222,6 +222,11 @@
 
 /obj/item/weapon/twohanded/dualsaber/attack(mob/target, mob/living/carbon/human/user)
 	..()
+	if(user.has_dna())
+		if(user.dna.check_mutation(HULK))
+			user << "<span class='warning'>You grip the blade too hard and accidentally close it!</span>"
+			unwield()
+			return
```
Comment (PKPenguin321) [inline code/game/objects/items/weapons/twohanded.dm]: i wasn't sure if i should put this before the parent call. i think as it is, the hulk can still get one attack off, but then the blade will close immediately after. hopefully this is fine.

**Comment by Supermichael777** [comment]
This nerf to hulks can probably be removed now that the reflect chance is not 100 percent.

File: code/game/objects/items/weapons/twohanded.dm (hunk 1 above)
Comment (Supermichael777) [inline code/game/objects/items/weapons/twohanded.dm]: oh yeah move the parent call to after

Comment (PKPenguin321) [inline code/game/objects/items/weapons/twohanded.dm]: moved it, this comment didn't close for some reason

**Pull request merged**